treeherder/etl/runnable_jobs.py

```diff
@@ -127,11 +128,14 @@ def _taskcluster_runnable_jobs(project, decision_task_id):
     if not decision_task_id:
         return ret
 
-    tc_graph_url = settings.TASKCLUSTER_TASKGRAPH_URL.format(task_id=decision_task_id)
+    tc_graph_url = settings.TASKCLUSTER_PUBLIC_ARTIFACTS_URL.format(
+        task_id=decision_task_id,
+        filename='runnable-jobs.json.gz'
```

Comment:
Since the filename is always the same, let's move it to `TASKCLUSTER_PUBLIC_ARTIFACTS_URL`